docs/kop.md

```diff
@@ -131,8 +131,38 @@ After you have installed the KoP protocol handler to Pulsar broker, you can rest
         This is another message
         ```
 
+### Run KoP in Docker
+
+KoP is a built-in component in StreamNative's `sn-pulsar` image, whose tag matches KoP's version. Take KoP 2.9.1.1 for example, you can run
```

Comment:
you can change it to the followings:

Take KoP 2.9.1.1 for example, you can execute the `docker compose up` command in the KoP project......